code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -175,8 +175,8 @@
 		/obj/item/stack/sheet/glass = 50,
 		/obj/item/stack/sheet/rglass = 50,
 		/obj/item/stack/cable_coil/cyborg = 50,
-		/obj/item/stack/rods = 15,
-		/obj/item/stack/tile/plasteel = 15
+		/obj/item/stack/rods = 30,
+		/obj/item/stack/tile/plasteel = 20

```

Comment:
All of these need to be <type>/cyborg where possible.

metal/cyborg
glass/cyborg
rglass/cyborg

I don't think there's a rods or plasteel cyborg versions though